### ISSUE TITLE
Error in foundry v10 development 2

### ISSUE THREAD

**Issue by strongpauly** [itamarcu/ZoomPanOptions#39]
```
foundry.js:711 LibWrapperPackageError: Error detected in module Zoom/Pan Options. It is likely this module has not been updated for FVTT V10 development 2.

This is *not* a libWrapper bug.

Find information about this module here: https://github.com/itamarcu/ZoomPanOptions
Report this issue here: https://github.com/itamarcu/ZoomPanOptions/issues

== Technical Details:
Detected by libWrapper.
Package ID= zoom-pan-options
Error= Can't wrap 'Canvas.prototype._constrainView', target does not exist or could not be found.

[Detected 1 package: zoom-pan-options]
    at new 🎁constructor (libWrapper-wrapper.js:111:11)
    at libWrapper-api.js:82:18
    at se (libWrapper-api.js:155:9)
    at Function.🎁register [as register] (libWrapper-api.js:525:14)
    at Object.fn (zoom-pan-options.js:412:14)
    at Function.#call (foundry.js:686:20)
    at Function.callAll (foundry.js:643:17)
    at Game.setupGame (foundry.js:7121:11)
    at Game._initializeGameView (foundry.js:8163:16)
    at Game._initializeView (foundry.js:8139:21)
    at Game.initialize (foundry.js:7026:16)
```

**Comment by itamarcu** [comment]
That must have been v1.7.0 which wasn't compatible yet.

Try the newly released v1.7.2 and reopen this if it still doesn't work!

**Comment by strongpauly** [comment]
Can confirm that error disappears in v1.7.2
